tests/src/test_cases/test_align_flex.c

```diff
@@ -105,6 +105,107 @@ void test_wrap_grow_min_width(void)
     TEST_ASSERT_EQUAL_SCREENSHOT("flex_wrap_grow_min_width.png");
 }
 
+static lv_obj_t * create_col_wrap(size_t count)
+{
+    lv_obj_t * col = lv_obj_create(lv_screen_active());
+    lv_obj_set_flex_flow(col, LV_FLEX_FLOW_COLUMN_WRAP);
+    lv_obj_set_flex_align(col, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_START, LV_FLEX_ALIGN_SPACE_EVENLY);
+
+    for(size_t i = 0; i < count; i++) {
+        lv_obj_t * obj = lv_obj_create(col);
+        lv_obj_set_style_pad_all(obj, 0, 0);
+        lv_obj_set_height(obj, 70 + (i % 3) * 20);
+        lv_obj_set_width(obj, 100);
+        lv_obj_set_style_bg_color(obj, lv_palette_main(LV_PALETTE_BLUE), 0);
+
+        lv_obj_t * label = lv_label_create(obj);
+        lv_label_set_text_fmt(label, "%lu", i);
```

Comment:
Please fix the CI issue by casting `i` or using a different type.

```suggestion
        lv_label_set_text_fmt(label, "%lu", (unsigned long) i);
```